src/index.html

```diff
@@ -7,8 +7,86 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo"></a>
+      <nav class="nav__bar">
+        <ul class="nav__ul">
+          <li class="nav__list">
```

Comment:
elements should be inside block (nav)

```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
```